core/app/modules/kit-library/assets/js/pages/preview/preview.js

```diff
@@ -118,7 +141,7 @@ export default function Preview( props ) {
 			<ItemHeader
 				model={ data }
 				buttons={ headersButtons }
-				centerColumn={ <PreviewResponsiveControls active={ activeDevice } onChange={ setActiveDevice } /> }
+				centerColumn={ <PreviewResponsiveControls active={ activeDevice } onChange={ setActiveDevice } kitName={ data.title } previewResponsiveControlsEvent={ ( device ) => previewResponsiveControlsEvent( device, data.title ) } /> }
```

Comment:
Rename `previewResponsiveControlsEvent` to be more generic

Reply:
onRespViewChange?